### ISSUE TITLE
Save file as . . . .options not working in 6.6.7 / 3.7.7

### ISSUE THREAD

**Issue by derimmer** [vasl-developers/vasl#1644]
The Save map as image and Save map contents to plain text file are not working in 6.6.7 and 3.7.7.

They do work with  3.7.6.

**Comment by BigAl737** [comment]
Hi Doug, I'm curious as to what caused this. I'm wondering if the issue might affect other modules I've made. Thanks.

**Comment by derimmer** [comment]
I don't know to be honest. With VASL we extend the two VASSAL classes ImageSaver and TextSaver. Our changes have only one purpose and that is to move the functionality from being trigged by a button on the toolbar to a line item on the VASL popop menu (potato masher guy). I don't know why we did that - probably to get more toolbar space. 

I was playing around with a request to have the Image option save As Seen By Opponent" in addition to the standard view. With Textsaver both options already existed. So maybe I accidently pushed some code changes before I had completed this work and that broke everything. I have since finished these changes and now everything seems to work in 668beta6. But that is still a bit of a guess. With so many VASSAL versions coming and going it is getting tough to know who is breaking what.
